src/main/java/software/amazon/smithy/lsp/syntax/Parser.java

```diff
@@ -813,15 +815,16 @@ private void enumMember(Syntax.Statement.Block parent) {
             Syntax.Ident name = ident();
             var enumMemberDef = new Syntax.Statement.EnumMemberDef(parent, name);
             enumMemberDef.start = start;
+            setEnd(enumMemberDef);
```

Comment:
What's the difference between setting the end two different times here, vs how it was before, just setting it once after the if?

Reply:
Nevermind, I get it - because the whitespace gets consumed.

Reply:
Can just add a comment here to explain why